src/components/Form.tsx

```diff
@@ -0,0 +1,18 @@
+export default function Form() {
```

Comment:
Would you consider to put on return type on the function to prevent you from doing stupid mistakes